src/app/operators/operators.component.ts

```diff
@@ -61,18 +63,38 @@ interface OperatorDocMap {
   ]
 })
 export class OperatorsComponent implements OnInit, AfterContentInit, OnDestroy {
-  @ViewChild(MatSidenav) _sidenav: MatSidenav;
-  public groupedOperators: OperatorDocMap;
-  public categories: string[];
   private _onDestroy = new Subject();
 
+  @ViewChild(MatSidenav) _sidenav: MatSidenav;
+  groupedOperators: OperatorDocMap;
+  categories: string[];
+  smallScreen$: Observable<boolean>;
+  operatorMenuGap$: Observable<number>;
```

Comment:
Technically speaking, I know these will be declared as public class members, but that's not the style I've seen in the project thus far.